### ISSUE TITLE
Malevolent Macro Malfunction (Maybe?)

### ISSUE THREAD

**Issue by jeffmeier** [statamic/v2-hub#350]
The following works well when written directly in a template (outputs processed markdown, *sans* `p` tags):

```
{{ something_interesting | markdown | strip_tags:p }}
```

But when I do it via macro, it doesn't (the output is raw, unprocessed markdown, as if no modifier at all):

```
# theme/settings/macros.yaml
interesting_macro:
  markdown: true
  strip_tags: p
```

```
{{# template.html #}}
{{ something_interesting | macro:interesting_macro }}
```

Despite the fact that *this* works as expected (outputs markdown *with* `p` tags):

```
# theme/settings/macros.yaml
interesting_macro:
  markdown: true
```

```
{{# template.html #}}
{{ something_interesting | macro:interesting_macro }}
```

I feel like it's possible I'm doing it wrong, so it might not be a bug. Any guidance?

P.S. sorry for alliteration...

**Comment by subpixelch** [comment]
Strip_Tags doesn't work at all in macros... no matter how formated

**Comment by jasonvarga** [comment]
Finally fixed for 2.0.10
